git_machete/client.py

```diff
@@ -1906,9 +1906,10 @@ def __sync_before_creating_pr(self) -> None:
         if current_branch not in self.managed_branches:
             self.add(current_branch)
 
-        if self.__git.is_ancestor_or_equal(current_branch, self.up_branch.get(current_branch)):
+        up_branch = self.up_branch.get(current_branch)
```

Comment:
What if `up_branch` is `None` (i.e. `current_branch` is a root branch)? The check whether parent branch exists happens in `create_github_pr` only after `__sync_before_creating_pr` is called. If there's indeed a bug here, pls add a regression test for this case as well (opening a PR for a root branch).